state/state.go

```diff
@@ -454,18 +454,18 @@ func (st *State) getLeaseClient(namespace string) (lease.Client, error) {
 		return nil, errors.Trace(err)
 	}
 
-	client, err := statelease.NewClient(statelease.ClientConfig{
-		Id:          st.leaseClientId,
+	store, err := statelease.NewStore(statelease.StoreConfig{
```

Comment:
Instead of having to add a new ModelUUID() method to environMongo, I think it's cleaner just to add a model-uuid attribute to StoreConfig. namespace and model-uuid are both used to make the lease key and having these together as attributes on the config struct makes most sense.  The Mongo abstraction should just continue to be the way to get to the database.

Reply:
Yeah, makes sense. I'll change that.